### PR TITLE
Release 1.0 for shelter app from Dev branch

### PR DESCRIPTION
**Pull request by guilbep**
# Describe your changes

Concerns the shelter app and the issues for the milestone release1.0: https://github.com/orgs/EPFL-ENAC/projects/13/views/8

# Related GitHub issues and pull requests

-  #207
- #197
- #232
- #241 
- #234 
- #231 
- #249

### CODE REVIEW

File: frontend/src/App.vue [hunk 1]
```diff
@@ -322,7 +364,43 @@ export default class App extends Vue {
   }
   get themeDark(): boolean {
     return false;
-    // return this.$store.getters["ConfigModule/themeDark"];
+  }
+
+  get shelterHelpers(): Record<string, string>[] {
```
Comment (blueur) [inline frontend/src/App.vue:369]: more precise ?
{title: string, icon?: string, type?: string, href?: string}

File: frontend/src/components/HelperCenter.vue [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+<style lang="scss">
+.pdf-card {
+  height: 80vh;
+  display: inline-block;
+  line-height: 0;
+}
+.default-card {
+  height: 300px;
+}
+// .v-dialog__content--active:has(> .notification-center),
```
Comment (blueur) [inline frontend/src/components/HelperCenter.vue:96]: comment ?

File: frontend/src/components/commons/logos.ts [hunk 3]
```diff
@@ -31,23 +31,23 @@ export const logos = [
...
-    urlDescription: "https://www.smartlivinglab.ch/fr/",
-    height: "60px",
-  },
-  {
-    imgPath: "/app_logo_new/Logo_FAR.webp",
-    label: "FAR",
-    urlDescription: "https://www.epfl.ch/labs/far/",
-    height: "60px",
-  },
+  // {
```
Comment (blueur) [inline frontend/src/components/commons/logos.ts:34]: comment ?

File: frontend/src/components/shelter_sustainability/CheckboxGroup.vue [hunk 4]
```diff
@@ -158,9 +164,13 @@ type CheckboxScore = Record<string, boolean>;
...
 .unhcr-expansion-panel {
   .v-expansion-panel-header {
+    display: grid;
+    grid-auto-columns: 1;
+    grid-auto-flow: column;
+    grid-template-columns: auto max-content min-content;
     .v-input--selection-controls {
-      margin-top: 0px;
-      padding-top: 0px;
+      // margin-top: 0px;
```
Comment (blueur) [inline frontend/src/components/shelter_sustainability/CheckboxGroup.vue:172]: comment ?

File: frontend/src/components/shelter_sustainability/InfoGroup.vue [hunk 5]
```diff
@@ -64,3 +64,33 @@ interface Info {
   description: string;
 }
 </script>
+
+<style lang="scss">
```
Comment (blueur) [inline frontend/src/components/shelter_sustainability/InfoGroup.vue:68]: scope ?

File: frontend/src/components/shelter_sustainability/RadioGroup.vue [hunk 6]
```diff
@@ -144,3 +162,83 @@ export default class RadioGroup extends Vue {
   }
 }
 </script>
+
+<style lang="scss">
```
Comment (blueur) [inline frontend/src/components/shelter_sustainability/RadioGroup.vue:166]: scope ?

File: frontend/src/views/shelter_sustainability/ShelterSustainabilityItem/Step1ProjectInformation.vue [hunk 7]
```diff
@@ -27,7 +27,7 @@
         <v-sheet v-if="localShelter" elevation="2" rounded>
           <v-container fluid>
             <v-row>
-              <v-col cols="4">
+              <!-- <v-col cols="4">
```
Comment (blueur) [inline frontend/src/views/shelter_sustainability/ShelterSustainabilityItem/Step1ProjectInformation.vue:30]: comment ?

File: frontend/src/views/shelter_sustainability/ShelterSustainabilityItem/Step10ReportCard.vue [hunk 8]
```diff
@@ -20,43 +20,245 @@
...
+              style="margin: -6px"
+            >
+              $mdiHumanMale
+            </v-icon>
+          </v-col>
+          <v-col class="align-center"
+            >{{ shelter.shelter_occupants }} intented occupants</v-col
+          >
+        </v-row>
+        <v-row>
```
Comment (blueur) [inline frontend/src/views/shelter_sustainability/ShelterSustainabilityItem/Step10ReportCard.vue:58]: v-for ?

File: frontend/src/views/shelter_sustainability/ShelterSustainabilityCompare.vue [hunk 9]
```diff
@@ -502,4 +576,12 @@ export default class ShelterSustainabilityCompare extends Vue {
   color: var(--c-grey);
   fill: var(--c-grey);
 }
+// @media print {
```
Comment (blueur) [inline frontend/src/views/shelter_sustainability/ShelterSustainabilityCompare.vue:579]: comment ?

**Review by blueur — CHANGES_REQUESTED** [review]

File: frontend/src/views/shelter_sustainability/ShelterSustainabilityCompare.vue (hunk 9 above)
Comment (guilbep) [reply frontend/src/views/shelter_sustainability/ShelterSustainabilityCompare.vue]: thanks

File: frontend/src/views/shelter_sustainability/ShelterSustainabilityItem/Step1ProjectInformation.vue (hunk 7 above)
Comment (guilbep) [reply frontend/src/views/shelter_sustainability/ShelterSustainabilityItem/Step1ProjectInformation.vue]: thanks

**Review by blueur — APPROVED** [review]

**Pull request merged**